src/views/lighttable.c

```diff
@@ -652,6 +675,33 @@ const dt_action_def_t dt_action_def_preview
       _action_elements_preview,
       NULL, TRUE };
 
+static float _action_process_infos(gpointer target, dt_action_element_t element, dt_action_effect_t effect,
+                                   float move_size)
```

Comment:
`const float move_size`

Reply:
I've fixed that, and the other `const` in the same function.
That said, @dterrahe : there's a lot of other place whith same function, where the const is not here... Is there a reason for that ? can I 'fix' them too ?

Reply:
> I've fixed that, and the other `const` in the same function. That said, @dterrahe : there's a lot of other place whith same function, where the const is not here... Is there a reason for that ? can I 'fix' them too ?

There is probably no good reasons. The const is often forgotten as not mandatory.  But I encourage all devs to fix them when they encounter some.